bin/saya/scripts/2_world_config.sh

```
@@ -0,0 +1,40 @@
+# Set environment variables for the world configuration
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)